compendium/modules/w12-assignment-photo.tex

```diff
@@ -82,56 +82,56 @@ \subsection{Uppgiften}
 
 
 \code{ImageEditor}-klassen ska använda sig av en Stack[Image] för att hantera historiken. 
-Läs dokumentationen här: \url{https://www.scala-lang.org/api/2.13.6/scala/collection/mutable/Stack.html}
+Läs dokumentationen här: \url{https://www.scala-lang.org/api/3.0.1/scala/collection/mutable/Stack.html}
```

Comment:
Better use latest so we go with the version flow: 
https://www.scala-lang.org/api/current/scala/collection/mutable/Stack.html